src/debugger.ts

```diff
@@ -151,28 +151,31 @@ export async function getDebugConfigurationFromCache(cache: CMakeCache, target:
   }
 
   if (!debuggerPathOverride) {
-    const clang_compiler_regex = /(clang[\+]{0,2})+(?!-cl)/gi;
     // Look for a debugger, in the following order:
-    const cpptoolsExtension = vscode.extensions.getExtension('ms-vscode.cpptools');
-    const cpptoolsDebuggerPath = cpptoolsExtension ? path.join(cpptoolsExtension.extensionPath, "debugAdapters", "lldb-mi", "bin", "lldb-mi") : undefined;
+    // 1. LLDB-MI
+    const clang_compiler_regex = /(clang[\+]{0,2})+(?!-cl)/gi;
     let clang_debugger_path = compiler_path.replace(clang_compiler_regex, 'lldb-mi');
```

Comment:
You might want to rename this to `mi_debugger_path` since it covers `lldb-mi`, `gdb`, and `lldb`  (not blocking, you can submit without this)